db/migrate/20150418193801_create_donations.rb

```diff
@@ -0,0 +1,16 @@
+class CreateDonations < ActiveRecord::Migration
+  def change
+    create_table :donations do |t|
+      t.string :name,             null: false, default: ''
+      t.string :email,            null: false, default: ''
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.